### ISSUE TITLE
Missing 1 required positional argument: 'task'

### ISSUE THREAD

**Issue by dcavadia** [nateraw/huggingpics#8]
Hello, im following the tutorial but when training the model it trows an error when using the PyTorch Lightning method

![image](https://user-images.githubusercontent.com/53955221/206601399-545cbca4-9e8b-4a9c-9e83-248c13aefb5d.png)
![image](https://user-images.githubusercontent.com/53955221/206601423-afb1f9ee-c9b7-47b8-ac15-ea8f4b984c20.png)

What could it be?

**Comment by nateraw** [comment]
Looks like torchmetrics updated and you have to provide a "task" now.

https://torchmetrics.readthedocs.io/en/stable/classification/accuracy.html#accuracy

If your classes list is >2, you should use "multiclass"

So wherever the `Accuracy` class is created, you do `Accuracy(task='multiclass')`

Will fix when I get the chance - thanks for the bug report!

**Comment by dcavadia** [comment]
oh thanks for the help it did solve the problem

**Comment by nateraw** [comment]
Going to reopen and leave it open til I issues a fix :)

**Comment by nateraw** [comment]
Ok, fixed!
